packages/menu-bar/test/overflow.test.js

```diff
@@ -462,4 +470,20 @@ describe('overflow', () => {
       expect(item.classList.contains('test-class-1')).to.be.true;
     });
   });
+
+  describe('performance', () => {
+    let menu, spy;
+
+    beforeEach(() => {
+      menu = fixtureSync('<vaadin-menu-bar></vaadin-menu-bar>');
+      spy = sinon.spy(menu, '_hasOverflow', ['get', 'set']);
```

Comment:
I wasn't able to properly spy on `offsetWidth` of the container. Also using `_hasOverflow` is only reasonable for initial render test but not for subsequent resizes, as in those cases it will be set at least twice: first by calling `__updateOverflow()` with empty array, and then with the actual items. But for this test IMO it's ok.